src/util/event/NumberGeneratorEvent.ts

```diff
@@ -0,0 +1,6 @@
+export class NumberGeneratorEvent{
+    args: number;
```

Comment:
replace args with some other name

Reply:
should be args, so it can be understand well that where and what are arguments